apps/ocpp/admin/charger/simulator.py

```
@@ -0,0 +1,136 @@
+        message = format_html(
+            ngettext(
+                "Created {count} simulator for the selected charge point. First simulator: {simulator}.",
+                "Created {count} simulators for the selected charge points. First simulator: {simulator}.",
+                total,
+            ),
+            count=total,
+            simulator=link,
+        )
+        self.message_user(request, message, level=messages.SUCCESS)
+        if total == 1:
+            detail_message = format_html(
```

Ruff (0.15.0)

[error] `format_html` may be undefined, or defined from star imports

(F405)